.github/workflows/release.yml

```diff
@@ -28,9 +28,12 @@ jobs:
     name: "Publish"
     needs: [release]
     if: ${{needs.release.outputs.releases_created}}
+    uses: linc-technologies/github/.github/workflows/ember_publish.yml@main
     permissions:
       contents: read
       id-token: write
```

Comment:
It would be amazing to not have these permissions defined in triplicate. Can we leverage the permissions set in the upstream CI manifest?